frontend/apps/web/app/(mgmt)/[account]/new/job/connect/page.tsx

```diff
@@ -144,156 +178,194 @@ export default function Page({ searchParams }: PageProps): ReactElement {
+                                    }
+                                  );
+                                } else {
+                                  form.setValue(
+                                    'sourceOptions',
+                                    {},
+                                    {
+                                      shouldDirty: true,
+                                      shouldTouch: true,
+                                      shouldValidate: true,
+                                    }
+                                  );
                                 }
-                              );
-                            }
-                          }}
-                          value={field.value}
-                        >
-                          <SelectTrigger
-                            className={cn(
-                              field.value ? undefined : 'text-muted-foreground'
-                            )}
-                          >
-                            <SelectValue
-                              ref={field.ref}
-                              placeholder="Select a source ..."
-                            />
-                          </SelectTrigger>
-                          <SelectContent>
-                            <ConnectionSelectContent
-                              postgres={postgres}
-                              mysql={mysql}
-                              mongodb={mongodb}
-                              dynamodb={dynamodb}
-                              mssql={mssql}
-                              newConnectionValue={NEW_CONNECTION_VALUE}
+                              }}
+                              value={field.value}
+                            >
+                              <SelectTrigger
+                                className={cn(
+                                  field.value
+                                    ? undefined
+                                    : 'text-muted-foreground'
+                                )}
+                              >
+                                <SelectValue
+                                  ref={field.ref}
+                                  placeholder="Select a source ..."
+                                />
+                              </SelectTrigger>
+                              <SelectContent>
+                                <ConnectionSelectContent
+                                  postgres={postgres}
+                                  mysql={mysql}
+                                  mongodb={mongodb}
+                                  dynamodb={dynamodb}
+                                  mssql={mssql}
+                                  newConnectionValue={NEW_CONNECTION_VALUE}
+                                />
+                              </SelectContent>
+                            </Select>
+                          </div>
+                          <div>
+                            <TestConnectionBadge
```

Comment:
I watched the loom video. I think this component itself looks fine, but the error and warnings states are really long and kind of stick out.
<img width="936" alt="image" src="https://github.com/user-attachments/assets/f62ebefe-d254-42bc-ac90-e61e86064179">
Is there any way we could make this smaller or have it appear on the line below?
Or perhaps just be an icon that has a hover tooltip?
What does this look like on smaller screen sizes?